build/build-static-binaries.sh

```diff
@@ -9,7 +9,13 @@ tags=${2-}
 source "$(dirname "${0}")"/build-common.sh
 
 time make STATIC=1 build
+# Build test binaries. Note that the acceptance binary will not be included in
+# the archive, but is instead uploaded directly by push-aws.sh.
+#
+# TODO(tamird,jordan): consider running race-enabled acceptance tests to
+# expose data races in the acceptance tests themselves.
 time make STATIC=1 testbuild PKG=./... TAGS="$tags acceptance"
+time make STATIC=1 testbuild PKG=./... TAGS="$tags acceptance" GOFLAGS=-race TESTBUILDPREFIX=race.
```

Comment:
Agreed. Moved to build-race-binaries in a new commit; PTAL and I'll squash if it looks good to you.